src/main/resources/spring/camel-context.xml

```diff
@@ -256,6 +256,37 @@
                     <bean ref="flagService" method="findByReportAnalysisOwnerAndReportAnalysisId(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.id}, ${header.pageBean}, ${header.sortBean})" />
                 </route>
             </get>
+            <get uri="/{id}/payload">
+                <route id="report-payload-download">
+                    <bean ref="analysisService" method="findByOwnerAndId(${header.${type:org.jboss.xavier.integrations.route.MainRouteBuilder.USERNAME}}, ${header.id})"/>
+                    <setHeader headerName="payloadname">
+                        <simple>${body.payloadName}</simple>
+                    </setHeader>
+                    <choice>
+                        <when>
+                            <simple>${body} != null &amp;&amp; ${body.payloadURL} != null</simple>
+                            <pollEnrich id="pollEnrich">
+                                <simple>aws-s3:xavier-dev?region={{S3_REGION}}&amp;accessKey={{S3_ACCESS_KEY_ID}}&amp;secretKey=RAW({{S3_SECRET_ACCESS_KEY}})&amp;deleteAfterRead=false&amp;fileName=${body.payloadURL}</simple>
+                            </pollEnrich>
+                            <setHeader headerName="Content-Disposition">
+                                <simple>${header.CamelAwsS3ContentDisposition}</simple>
+                            </setHeader>
+                            <setHeader headerName="Content-Type">
```

Comment:
You can replace `Content-Type` with [`CustomizedMultipartDataFormat.CONTENT_TYPE`](https://github.com/project-xavier/xavier-integration/blob/master/src/main/java/org/jboss/xavier/integrations/route/dataformat/CustomizedMultipartDataFormat.java#L34)